tools.yml

```diff
@@ -1053,7 +1053,9 @@ tools:
   toolshed.g2.bx.psu.edu/repos/iuc/anndata_export/anndata_export/.*:
     cores: 4
   toolshed.g2.bx.psu.edu/repos/iuc/anndata_import/anndata_import/.*:
-    mem: input_size * 10
+    rules:
+    - id: anndata_import_mem_rule
+      mem: input_size * 10
```

Comment:
I disagree with this change... What is the point of doing what you can do in one line using three lines?

Reply:
`mem: input_size * 10` is not what we call a rule. The [TPV documentation](https://total-perspective-vortex.readthedocs.io/en/latest/topics/tpv_by_example.html#rules) states that

> Rules provide a means by which to conditionally change entity requirements.

The rule is the construct that you put under the `rules` key. It has:
- an id, that allows overriding it,
- a condition to evaluate (the `if:` key, that defaults to `if: True` when unspecified),
- a `fail:` clause for erroring out when the condition evaluates to false,
- all other usual keys for the entity (in this case the tool), that will be taken into consideration only if the condition evaluates to true

In this case we want `mem: input_size * 10` to hold **always**, so we do not need a rule.

Reply:
I forgot to mention the most important past: because of [all that has been stated in the previous message](https://github.com/galaxyproject/tpv-shared-database/pull/24#discussion_r1233645894), please revert to 

```
  toolshed.g2.bx.psu.edu/repos/iuc/anndata_import/anndata_import/.*:
    mem: input_size * 10
```

Reply:
Oh ok didn't know that. Thanks for explaining.

Reply:
> Oh ok didn't know that. Thanks for explaining.

Sorry for the further explanation that I provided a minute ago, I did not read this :sweat_smile:

Reply:
Thanks for pointing out. I think now we are finally ready :)